runtime/kusama/src/lib.rs

```diff
@@ -1602,7 +1602,10 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	pallet_nomination_pools::migration::v2::MigrateToV2<Runtime>,
+	(
+		pallet_nomination_pools::migration::v2::MigrateToV2<Runtime>,
+		pallet_staking::migrations::v10::MigrateToV10<Runtime>,
```

Comment:
```suggestion
		pallet_staking::migrations::v10::MigrateToV10<Runtime>,
```

Can this be removed now? Same for Westend.

Reply:
yeah, but I rather leave that to the release team. Ideally they should have a process around it. All migrations that I write have a guard and will not be re-executed.